pkgdb/.clang-format

```diff
@@ -79,10 +79,10 @@ SpacesInSquareBrackets: false
 Standard: Auto
 TabWidth: 2
 UseTab: Never
-InsertBraces: true
+# InsertBraces: true
 EmptyLineBeforeAccessModifier: Always
 EmptyLineAfterAccessModifier: Always
-BreakBeforeConceptDeclarations: Always
+BreakBeforeConceptDeclarations: true
```

Comment:
I don't really care about changes here, but I'll either let @aakropotkin  review them, or I can approve if you drop them

Reply:
clang-format complained that `BreakBeforeConceptDeclarations` was not a boolean, so `true` it is i guess

Reply:
Hmm from the docs it seems `Always` should be okay? https://clang.llvm.org/docs/ClangFormatStyleOptions.html#breakbeforeconceptdeclarations

What about `InsertBraces`?

Reply:
didn't find it at all

We are also using clang-format `16.x.y` that documentation is about version `18`

Reply:
Hmm I'm seeing it in 16.0.0? Is that still the wrong version?

https://releases.llvm.org/16.0.0/tools/clang/docs/ClangFormatStyleOptions.html#breakbeforeconceptdeclarations

https://releases.llvm.org/16.0.0/tools/clang/docs/ClangFormatStyleOptions.html#insertbraces

Reply:
correction: pre-commit-hook uses 14.0.6 (/nix/store/0r07ld5n1490rca3n2kldhca5ii5saz5-clang-tools-14.0.6/bin/clang-format) for some reason.
in that version [breakbeforeconceptdeclarations](https://releases.llvm.org/14.0.0/tools/clang/docs/ClangFormatStyleOptions.html#breakbeforeconceptdeclarations) was still a boolean 
<img width="571" alt="image" src="https://github.com/flox/flox/assets/7040031/ec1dcff0-005a-4b33-a344-1ff3f7e1072c">

and insertbraces does not exist.

let me see if i can update that

Reply:
ok we were explicitly using `clang-tools_16` in the shell.
I updated the pre-commit config to use that package as well and restored the .clang-format